README.md

```diff
@@ -86,9 +86,23 @@ Unix tools:
 
 [Google Chrome]: https://www.google.com/chrome/
 
+# iTerm2
+* [iterm2] is a replacement for Terminal with theming and other advanced features
+
+# Fonts
```

Comment:
What's this used for?

Reply:
which, iterm or fonts? the font is used to support additional characters (ligatures) in IDEs as well as powerline characters on the command-line: https://betterwebtype.com/articles/2020/02/13/5-monospaced-fonts-with-cool-coding-ligatures/
powerline characters: https://fmacedoo.medium.com/oh-my-zsh-with-powerline-fonts-pretty-simple-as-you-deserve-fbe7f6d23723

iterm2 supports more configurability on keyboard mapping, so you can re-enable home/end, etc on a mac